bundles/org.eclipse.equinox.p2.director.app/META-INF/MANIFEST.MF

```diff
@@ -2,12 +2,13 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.p2.director.app;singleton:=true
-Bundle-Version: 1.3.300.qualifier
+Bundle-Version: 1.3.400.qualifier
 Bundle-Activator: org.eclipse.equinox.internal.p2.director.app.Activator
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Export-Package: org.eclipse.equinox.internal.p2.director.app;x-internal:=true
-Require-Bundle: org.eclipse.equinox.common;bundle-version="[3.5.0,4.0.0)"
+Require-Bundle: org.eclipse.equinox.common;bundle-version="[3.5.0,4.0.0)",
+ org.eclipse.equinox.p2.publisher;bundle-version="1.9.100"
```

Comment:
Please specify a range.  The thing you are using was added for 1.9.100 so this is not an accurate version.

Reply:
Why 3.0.0 and not 2.0.0?  The range should specify an accurate and meaningful lower bound.  The things from JRE Action that you are using are first available in 1.9.200 so this bundle cannot work with versions 1.9.100 or 1.9.0.